From 9d586c774309f46b925839060902b786711764fa Mon Sep 17 00:00:00 2001
From: Ankit Agarwal <ankit.agarwal@intel.com>
Date: Fri, 2 Aug 2024 21:14:48 +0530
Subject: [PATCH] Updated media c2 hal from hidl to aidl.

Updated android.hardware.media.c2 hal from hidl V1.0 to aidl V1.

Tests: Prepared EB and its booted successfully.

Tracked-On: OAM-122196
Signed-off-by: Ankit Agarwal <ankit.agarwal@intel.com>
---
 groups/codec2/true/product.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/codec2/true/product.mk b/groups/codec2/true/product.mk
index ea15750..0e40117 100755
--- a/groups/codec2/true/product.mk
+++ b/groups/codec2/true/product.mk
@@ -18,7 +18,7 @@ PRODUCT_COPY_FILES += \
 
 PRODUCT_PACKAGES += \
     libmfx_c2_components_hw \
-    hardware.intel.media.c2@1.0-service
+    android.hardware.media.c2-service.intel
 
 BOARD_HAVE_MEDIASDK_CODEC2 := true
 {{/enable_msdk_c2}}
-- 
2.45.2

